content/agent/basic_agent_usage/_index.md

```diff
@@ -1,4 +1,96 @@
 ---
-title: Basic Agent Usage
-external_redirect: /agent/basic_agent_usage/amazonlinux/
+title: Basic Agents Usage
+kind: documentation
+aliases:
+    - /guides/basic_agent_usage/
+    - /agent/faq/where-is-the-configuration-file-for-the-agent/
+    - /agent/faq/log-location
 ---
+
+{{< partial name="platforms/platforms.html" >}}
+
+## Configuration management tools
+
+Manage the Datadog Agent and [integrations](/integrations) using configuration management tools:
+
+### Chef
+* [Chef Github project](https://github.com/DataDog/chef-datadog)
+* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
+
+### Puppet
+* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
+* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
```

Comment:
follow the same pattern as i mentioned for chef

Reply:
👌